x/evm/keeper/attest.go

```diff
@@ -58,13 +61,14 @@ func (c *consensusPower) consensus() bool {
 		===
 		3 * sum >= totalPower * 2
 	*/
-	return c.runningSum.Mul(sdk.NewInt(3)).GTE(
-		c.totalPower.Mul(sdk.NewInt(2)),
+	return c.runningSum.Mul(sdkmath.NewInt(3)).GTE(
+		c.totalPower.Mul(sdkmath.NewInt(2)),
 	)
 }
 
-func (k Keeper) attestRouter(ctx sdk.Context, q consensus.Queuer, msg consensustypes.QueuedSignedMessageI) (err error) {
-	logger := k.Logger(ctx).WithFields(
+func (k Keeper) attestRouter(ctx context.Context, q consensus.Queuer, msg consensustypes.QueuedSignedMessageI) (err error) {
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+	logger := k.Logger(sdkCtx).WithFields(
```

Comment:
@aleem1314 @vishal-kanna 

Please start using `liblog.FromSDKLogger(k.Logger(SdkCtx)` in the future. There are still ocurrances here without.